libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -135,7 +135,15 @@ class DropInNavigationView @JvmOverloads constructor(
          */
         attachCreated(
             BackPressManager(navigationContext),
-            MapCoordinator(navigationContext, binding.mapView),
+            MapCoordinator(
+                navigationContext,
+                binding.mapView,
+                listOf(
+                    binding.guidanceLayout,
+                    binding.actionListLayout,
+                    binding.infoPanelLayout
```

Comment:
Skipping Road Name and Speed Limit layouts for now. We can added them if needed.

Reply:
Could you explain more about the need to skip it for now? We do need the camera viewport edge insets to take speed limit and road name label into account. I think we should add it.

Reply:
I was comparing this with Google Maps and noticed that they use only top and bottom overlays to adjust their viewport. I've been using Google Maps for many years and I never saw a problem with this approach, hence the above.
This is entirely up to you. Would you like me to add it or should we leave it as it is?